README.md

```diff
@@ -4,20 +4,26 @@ Lightweight port of [VDKQueue](https://github.com/bdkjones/VDKQueue).
 SKQueue is a simple and efficient Swift wrapper around kernel queues (kQueues). It allows you to watch a file or folder for changes and be notified when they occur. See [VDKQueue](https://github.com/bdkjones/VDKQueue) for more info.
 
 ## Example
-    class SomeClass: SKQueueDelegate {
-        func receivedNotification(queue: SKQueue, _ notificationName: SKQueueNotificationString, forPath path: String) {
-            print("\(notificationName.rawValue) @ \(path)")
-        }
+```swift
+class SomeClass: SKQueueDelegate {
+    func receivedNotification(queue: SKQueue, _ notificationName: SKQueueNotificationString, forPath path: String) {
+        print("\(notificationName.rawValue) @ \(path)")
```

Comment:
Awesome with the syntax highlighting. I would prefer you do a separate PR with these changes though, just to keep the commit history as clean as possible.